db/migrate/20220905064408_add_index_on_network_and_data_centers_id_to_dc_stats.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexOnNetworkAndDataCentersIdToDcStats < ActiveRecord::Migration[6.1]
+  def change
+    add_index :data_center_stats, [:network, :data_center_id]
```

Comment:
I suppose we want to have unique values in this scope, maybe it would be good to add unique constraint on the db level?